prometheus/promhttp/delegator.go

```diff
@@ -76,6 +76,10 @@ func (r *responseWriterDelegator) Write(b []byte) (int, error) {
 	return n, err
 }
 
+func (r *responseWriterDelegator) Unwrap() http.ResponseWriter {
```

Comment:
Do you mind adding quick comment on this public method similar to your PR description?